static/app/components/profiling/flamegraph/flamegraph.tsx

```diff
@@ -368,74 +369,71 @@ function Flamegraph(): ReactElement {
     [flamegraph, flamegraphCanvas, flamegraphTheme, xAxis]
   );
 
-  const uiFramesView = useMemoWithPrevious<CanvasView<UIFrames> | null>(
-    _previousView => {
-      if (!flamegraphView || !flamegraphCanvas || !uiFrames) {
-        return null;
-      }
+  const uiFramesView = useMemo<CanvasView<UIFrames> | null>(() => {
+    if (!flamegraphView || !flamegraphCanvas || !uiFrames) {
+      return null;
+    }
 
-      const newView = new CanvasView({
-        canvas: flamegraphCanvas,
-        model: uiFrames,
-        mode: 'stretchToFit',
-        options: {
-          inverted: flamegraph.inverted,
-          minWidth: uiFrames.minFrameDuration,
-          barHeight: 10,
-          depthOffset: 0,
-          configSpaceTransform:
-            xAxis === 'transaction'
-              ? new Rect(flamegraph.profile.startedAt, 0, 0, 0)
-              : undefined,
-        },
-      });
+    const newView = new CanvasView({
+      canvas: flamegraphCanvas,
+      model: uiFrames,
+      mode: 'stretchToFit',
+      modelConfigSpace: flamegraph.configSpace,
+      options: {
+        inverted: flamegraph.inverted,
+        minWidth: uiFrames.minFrameDuration,
+        barHeight: 10,
+        depthOffset: 0,
+        configSpaceTransform:
+          xAxis === 'transaction'
+            ? new Rect(-flamegraph.profile.startedAt, 0, 0, 0)
```

Comment:
why is this negative?

Reply:
Doest have to be, I actually think we shouldnt even use the startedAt because I dont think it ever did what I thought it did, but I need to audit that in a separate PR